### ISSUE TITLE
Make primary colors customisable

### ISSUE THREAD

**Issue by devmount** [devmount/third-stats#406]
**Is your feature request related to a problem? Please describe.**
Primary colors for received and sent messages in ThirdStats are currently hardcoded to blue and pink.

**Describe the solution you'd like**
Provide additional add-on options to customize those two colors to provide a less opinionated experience. Default should be the current ThirdStats colors.

**Describe alternatives you've considered**
Maybe there is a way to somehow get accent colors from the currently used theme and take them as default colors.

**Additional context**
https://mstdn.io/@t3kk@mastodonapp.uk/110112482422893318

**Comment by Wilker-uwu** [comment]
would be cool if it were to follow whatever is Thunderbird's currently configured theme for its color set if possible \^\^

**Comment by devmount** [comment]
@Wilker-uwu Agreed, that would be nice. And I think it's now possible to read theme colors by using the Theme API: https://webextension-api.thunderbird.net/en/stable/theme.html#themecolor
